Modules/_bisectmodule.c

```diff
@@ -129,7 +141,8 @@ _bisect_insort_right_impl(PyObject *module, PyObject *a, PyObject *x,
             return NULL;
     }
     else {
-        result = PyObject_CallMethod(a, "insert", "nO", index, x);
+        bisect_state *state = get_bisect_state(module);
+        result = _PyObject_CallMethod(a, state->str_insert, "nO", index, x);
```

Comment:
You could call the `insert` method with `PyObject_Vectorcall` to make it even faster and avoid variadic args overhead.

Reply:
Thanks nice suggestion :)

Reply:
@kumaraditya303 
In this phase, I decide not to apply vectorcall for this change.
It will increase the complexity of implementation due to callable checking and conversion index into index_obj :)
Making `_PyObject_CallMethod` to use vectorcall aggressive will be more valuable than changing the implementation in this line.